docs/src/app/components/pages/page-with-nav.jsx

```diff
@@ -68,30 +73,34 @@ let PageWithNav = React.createClass({
         <div style={this.prepareStyles(styles.content)}>
           {this.props.children}
         </div>
-        <div style={this.prepareStyles(styles.secondaryNav)}>
-          <Menu
-            ref="menuItems"
-            zDepth={0}
-            menuItems={this.props.menuItems}
-            selectedIndex={this._getSelectedIndex()}
-            onItemTap={this._onMenuItemClick} />
-        </div>
+        <Menu
+          autoWidth={false}
+          onItemTouchTap={this._handleItemTouchTap}
+          openDirection="bottom-right"
+          style={styles.secondaryNav}
+          valueLink={{
+            value: this.state.selectedIndex,
+            requestChange: this._handleUpdateSelectedIndex,
+          }}
+          width={styles.secondaryNav.width}
+          zDepth={0}>
+          {this.props.menuItems.map((item, index) => {
+            return (<MenuItem
+              key={index}
+              onTouchTap={() => { this.history.pushState(null, item.route); }}
```

Comment:
There is a depreciation notice `Warning: [history] pushState is deprecated; use push instead`.
Can we use `this.history.push(item.route);` instead?